src/epcpm/parameterstointerface.py

```diff
@@ -303,21 +305,25 @@ def interface_variable_name(self):
         model = maybe_model
         model_variable = f"sunspecInterface.model{model.id}"
 
-        return f"&{model_variable}.{parameter.abbreviation}"
-
+        if reference == True:
+            return f"&{model_variable}.{parameter.abbreviation}"
+        else:
+            return f"{model_variable}.{parameter.abbreviation}"
 
 @builders(epcpm.sunspecmodel.DataPointBitfieldMember)
 @attr.s
 class DataPointBitfieldMember:
     wrapped = attr.ib()
     parameter_uuid_finder = attr.ib()
 
-    def interface_variable_name(self):
+    def interface_variable_name(self, reference=True):
         parameter = self.parameter_uuid_finder(self.wrapped.parameter_uuid)
 
         uuid_ = str(parameter.uuid).replace("-", "_")
-        return f"&interfaceItem_variable_{uuid_}"
-
+        if reference == True:
+            return f"&interfaceItem_variable_{uuid_}"
+        else:
+            return f"interfaceItem_variable_{uuid_}"
```

Comment:
Consider replacing with arguably not easier to read one liner:
`return f"{'&' if reference else ''}interfaceItem_variable_{uuid_}"`